LEAF_Request_Portal/js/formGrid.js

```diff
@@ -586,20 +558,20 @@ var LeafFormGrid = function(containerID, options) {
         renderVirtualHeader();
     }
 
-        /**
+    /**
      * @memberOf LeafFormGrid
      */
-         function announceResults(){
-             var term = $('[name="searchtxt"]').val();
-
-             if(currentData.length == 0) {
-                 $('.status').text('No results found for term ' + term);
-         }else{
-                 $('.status').text('Search results found for term ' + term + ' listed below');
-            }
+     function announceResults(){
+         var term = $('[name="searchtxt"]').val();
```

Comment:
var or let?

Reply:
I must have touched the spacing and now it's mine.  I can get this one though.